src/test/kotlin/org/rust/cargo/util/TestUnitTestRustcCacheService.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.util
+
+import com.intellij.util.containers.orNull
+import org.rust.cargo.toolchain.impl.RustcVersion
+import java.util.*
+import java.util.concurrent.ConcurrentHashMap
+
+class TestUnitTestRustcCacheService : UnitTestRustcCacheService {
```

Comment:
Let's move this implementation to production code as well. Since test code of the plugin is not included to plugin artifact, any other plugin with dependency on Rust plugin (for example, EduTools) will get `ClassNotDefFoundError` during test evaluation